### PR TITLE
[loader] SDF support

### PR DESCRIPTION
**Pull request by nim65s**
This is a proposition to add support for SDF models in pinocchio >= 2.9.2 in `RobotLoader`

### CODE REVIEW

**Review by cmastalli — COMMENTED** [review]
Should we always give priority to build with the URDF model? 

I suspect that might be cases where we want to build through the SDF model. If so, the loader needs to reflect the wish of giving specific priorities.

A part of this comment, the rest looks fine for me.

**Comment by nim65s** [comment]
I was assuming that we will never have a model with both an URDF model and a SDF model

File: python/example_robot_data/robots_loader.py [hunk 1]
```diff
@@ -50,19 +50,29 @@ class RobotLoader(object):
...
 
     def __init__(self):
-        urdf_path = join(self.path, self.urdf_subpath, self.urdf_filename)
-        self.model_path = getModelPath(urdf_path, self.verbose)
-        self.urdf_path = join(self.model_path, urdf_path)
-        self.robot = RobotWrapper.BuildFromURDF(self.urdf_path, [join(self.model_path, '../..')],
-                                                pin.JointModelFreeFlyer() if self.free_flyer else None)
+        if self.urdf_filename:
+            df_path = join(self.path, self.urdf_subpath, self.urdf_filename)
+            builder = RobotWrapper.BuildFromURDF
```
Comment (cmastalli) [inline python/example_robot_data/robots_loader.py:65]: I would add a warning message if the developer has defined `sdf_filename`. I would include the following code:
```python
  if self.sdf_filename:
    print("Warning: we are building the robot model from the URDF")
    print("If you wish to build the model from the SDF", then do not define the 'urdf_filename'")
```

**Comment by proyan** [comment]
ping :)
Could we please finish this PR, so that #100 could also be handled?

File: python/example_robot_data/robots_loader.py [hunk 2]
```diff
@@ -50,19 +50,31 @@ class RobotLoader(object):
...
+            if self.sdf_filename:
+                raise AttributeError("Please choose between URDF *or* SDF")
+            df_path = join(self.path, self.urdf_subpath, self.urdf_filename)
+            builder = RobotWrapper.BuildFromURDF
+        else:
+            df_path = join(self.path, self.sdf_subpath, self.sdf_filename)
+            try:
+                builder = RobotWrapper.BuildFromSDF
+            except AttributeError:
+                raise ImportError("Building SDF models require pinocchio >= 2.9.2")
```
Comment (proyan) [inline python/example_robot_data/robots_loader.py:73]:
```suggestion
                raise ImportError("Building SDF models require pinocchio >= 2.9.1")
```

Comment (jcarpent) [reply python/example_robot_data/robots_loader.py:73]: I would suggest using Pinocchio >= 3.0.0. 
This is much more simple, as 2.9.x won't be released on Conda, APT, etc.

Comment (nim65s) [reply python/example_robot_data/robots_loader.py:73]: Ok for 3. It's a little lie, but it won't hurt users who have 2.9 :)

**Pull request merged**